lib/rules/block-indentation.js

```diff
@@ -21,7 +21,7 @@
 
  <div>
   <p>{{t "greeting"}}</p>
- </div>
+  </div>
```

Comment:
This looks wrong?

Reply:
That's thé purpose. It says Bad example but It's actually a  good one 😅